xml/System.Security.Cryptography/RSAEncryptionPadding.xml

```diff
@@ -288,7 +288,11 @@
       <Docs>
         <summary>Gets the hash algorithm used in conjunction with the <see cref="F:System.Security.Cryptography.RSAEncryptionPaddingMode.Oaep" /> padding mode. If the value of the <see cref="P:System.Security.Cryptography.RSAEncryptionPadding.Mode" /> property is not <see cref="F:System.Security.Cryptography.RSAEncryptionPaddingMode.Oaep" />, <see cref="P:System.Security.Cryptography.HashAlgorithmName.Name" /> is <see langword="null" />.</summary>
         <value>The hash algorithm.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+If <xref:System.Security.Cryptography.RSAEncryptionPadding.Mode> is not <xref:System.Security.Cryptography.RSAEncryptionPaddingMode.Oaep>, then <xref:System.Security.Cryptography.HashAlgorithmName.Name> will be `null`.
```

Comment:
This is already in the summary. Should we remove it from there and leave it just on the remarks?

Reply:
I removed it from the summary.